diff --git a/Python day 1 to 7/Day2/day2part1.py b/Python day 1 to 7/Day2/day2part1.py
--- Python day 1 to 7/Day2/day2part1.py
+++ Python day 1 to 7/Day2/day2part1.py
@@ -18,5 +18,30 @@
     'year': year,
     'is_married': is_married,
     'is_true': is_true,
     'is_light_on': is_light_on
-} 
\ No newline at end of file
+}
+
+print(person_info['city'])
+
+#Day 2: 30 Days of python programming
+First_name = 'John'
+Last_name = 'Doe'
+full_name = First_name + ' ' + Last_name
+country = 'USA'
+city = 'New York'
+age = 30
+year = 2023
+is_married = False
+is_true = True
+is_light_on = True
+person_info = {
+    'first_name': First_name,
+    'last_name': Last_name,
+    'country': country,
+    'city': city,
+    'age': age,
+    'year': year,
+    'is_married': is_married,
+    'is_true': is_true,
+    'is_light_on': is_light_on
+} 
